src/main/java/com/odysseusinc/arachne/datanode/dto/serialize/ExpressionDeserializer.java

```diff
@@ -0,0 +1,27 @@
+package com.odysseusinc.arachne.datanode.dto.serialize;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.JsonToken;
+import com.fasterxml.jackson.core.TreeNode;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import java.io.IOException;
+import java.util.Objects;
+
+public class ExpressionDeserializer extends JsonDeserializer<String> {
+	@Override
+	public String deserialize(JsonParser parser, DeserializationContext context) throws IOException, JsonProcessingException {
```

Comment:
Some comment regarding logic would be very appreciated

Reply:
In older Atlas versions expression was a `String` containing json inside, but starting from version *2.8* it becomes an object. `ExpressionDeserializer` is required to remain consistent w/o changing internal logic of DataNode.